### PR TITLE
Pst 162

### PR DESCRIPTION
**Pull request by basaks**
This `createenv.sh` script should be kept uptodate with any changing system requirement

### CODE REVIEW

**Comment by basaks** [comment]
The `passive-siesmic/sc3/createenv.sh` should be run in the seiscomp3 aws image as the following:

```
$ cd ~
$ git clone https://github.com/GeoscienceAustralia/passive-seismic
$ ./passive-seismic/sc3/createenv.sh
```
This will install everything we need for the `seismic` virtualenv for python. Run tests like:

```
$ workon seismic
$ mpirun -n 4 pytest passive-seismic/tests/test_pyasdf.py
$ pytest passive-seismic/tests
```

File: sc3/createenv.sh [hunk 1]
```diff
@@ -64,13 +64,13 @@ git clone https://github.com/h5py/h5py.git && \
...
 # Setup virtualenv
+echo '' >> $HOME/.bashrc
 echo "source /usr/bin/virtualenvwrapper.sh" >> $HOME/.bashrc
 source $HOME/.bashrc
 
 mkvirtualenv --system-site-packages seismic
 
 echo "Installing passive seismic software....."
 workon seismic && \
-    git clone https://github.com/GeoscienceAustralia/passive-seismic && \
```
Comment (rh-downunder) [inline sc3/createenv.sh]: Was this dropped unintentionally..?

Comment (rh-downunder) [reply sc3/createenv.sh]: I see why. Looks great!

Comment (basaks) [reply sc3/createenv.sh]: Thanks @rh-downunder for the review. Merging.

**Pull request merged**